recipes/scipy/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version="0.17.0" %}
+
+package:
+  name: scipy
+  version: {{ version }}
+
+source:
+  fn: scipy-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/scipy/scipy-0.17.0.tar.gz
+  md5: 5ff2971e1ce90e762c59d2cd84837224
+
+build:
+  number: 0
+  features:
+    - openblas
+
+requirements:
+  build:
+    - python
+    - numpy     x.x
+    - openblas
+  run:
+    - python
+    - numpy     x.x
+    - openblas
+    - libgfortran 3.0*
+
+test:
+  requires:
+    - nose
+  imports:
+    - scipy
+    - scipy.integrate.vode
+    - scipy.linalg._fblas
+    - scipy.linalg._flapack
+    - scipy.sparse.linalg.eigen.arpack._arpack
+    - scipy.sparse.linalg.isolve._iterative
+
+about:
+  home: http://www.scipy.org/
+  license: BSD
```

Comment:
IIRC there is no actual umfpack code contained in scipy itself, but it will try to import scikit-umfpack (which depends on umfpack).  If the import succeeds then umfpack is used at runtime.